storm-client/src/jvm/org/apache/storm/StormSubmitter.java

```diff
@@ -106,27 +106,30 @@ public static boolean validateZKDigestPayload(String payload) {
      * @param name        the name of the topology to push credentials to.
      * @param topoConf    the topology-specific configuration, if desired. See {@link Config}.
      * @param credentials the credentials to push.
+     * @return whether the pushed credential collection fullCreds is non-empty. Return false if empty.
```

Comment:
`fullCreds` doesn't really mean anything, image anyone changed the variable name

Reply:
Sure. Fixed

Reply:
only one Travis build failed. Should be irrelevant.